linalg/symmat.cpp

```diff
@@ -58,6 +58,20 @@ DenseSymmetricMatrix &DenseSymmetricMatrix::operator=(real_t c)
    return *this;
 }
 
+DenseSymmetricMatrix &DenseSymmetricMatrix::operator=(const DenseSymmetricMatrix
+                                                      &m)
+{
+   SetSize(m.height);
+
+   const int hw = m.GetStoredSize();
+   for (int i = 0; i < hw; i++)
+   {
+      data[i] = m.data[i];
```

Comment:
`Memory::operator` functions read:
>    /// Array subscript operator for host memory.
   inline T &operator[](int idx);
>   /// Array subscript operator for host memory, const version.
   inline const T &operator[](int idx) const;
   
   will this only be evaluated on host ? Would `Memory::copyTo` be a better option ? (I am not familiar with this piece of code so I am asking to understand this better. Thanks.).

Reply:
It is copied from `DenseMatrix` implementation, where simply we do not care at the moment as anything else than host is not supported 😬

Reply:
I am not sure, I just essentially reproduced the code used for `DenseMatrix`'s copy operator (see [here](https://github.com/mfem/mfem/blob/7c296d00d8a770e9f569e4db9d7e6d415902a886/linalg/densemat.cpp#L668-L679)) but with `GetStoredSize()` instead.